src/main/webapp/style.css

```diff
@@ -145,9 +144,23 @@ header {
   background-color: white;
   border: none;
   cursor: pointer;
+  height: 37px;
+  max-height: 37px;
```

Comment:
You probably don't need max-height if it's the same as height. Also, this looks similar to logic that I had you remove in the past, and use line-height, font-size, and padding instead. Is that doable?